cli/command/system/info.go

```diff
@@ -143,6 +143,11 @@ func prettyPrintServerInfo(dockerCli command.Cli, info types.Info) []error {
 	fmt.Fprintln(dockerCli.Out(), "  Paused:", info.ContainersPaused)
 	fmt.Fprintln(dockerCli.Out(), "  Stopped:", info.ContainersStopped)
 	fmt.Fprintln(dockerCli.Out(), " Images:", info.Images)
+	if info.Builder == types.BuilderV1 || info.Builder == "" {
```

Comment:
What do you think about this?
```go
builder := "Unknown"
switch info.Builder{
case "", types.BuilderV1:
    builder = "Legacy"
case types.BuilderBuildKit:
    builder = "Buildkit"
default:
    errs = append(errs, fmt.Errorf("unknown builder type %q", info.Builder)
}
fmt.Fprintln(dockerCli.Out(), " Default Builder:", builder)
```

Reply:
I agree that looks much cleaner, switched to that 😄